src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/ThreadPoolConfiguration.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.task.TaskExecutor
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor
+
+@Configuration
+class ThreadPoolConfiguration {
+  @Bean("threadPoolTaskExecutor")
+  fun taskExecutor(): TaskExecutor =
+    ThreadPoolTaskExecutor().apply {
+      corePoolSize = 200
```

Comment:
I took these values from the medium post mentioned in the PR description - we can tweak these as necessary